examples/vision/maml_miniimagenet.py

```diff
@@ -62,9 +68,39 @@ def main(
     train_dataset = l2l.data.MetaDataset(train_dataset)
     valid_dataset = l2l.data.MetaDataset(valid_dataset)
     test_dataset = l2l.data.MetaDataset(test_dataset)
-    train_generator = l2l.data.TaskGenerator(dataset=train_dataset, ways=ways, tasks=20000)
-    valid_generator = l2l.data.TaskGenerator(dataset=valid_dataset, ways=ways, tasks=1024)
-    test_generator = l2l.data.TaskGenerator(dataset=test_dataset, ways=ways, tasks=1024)
+
+    train_transforms = [
+        l2l.data.transforms.NWays(train_dataset, ways),
+        l2l.data.transforms.KShots(train_dataset, 2*shots),
+        l2l.data.transforms.LoadData(train_dataset),
+        l2l.data.transforms.RemapLabels(train_dataset),
+        l2l.data.transforms.ConsecutiveLabels(train_dataset),
+    ]
+    train_tasks = l2l.data.TaskDataset(train_dataset,
+                                       task_transforms=train_transforms,
+                                       num_tasks=20000)
+
+    valid_transforms = [
+        l2l.data.transforms.NWays(valid_dataset, ways),
+        l2l.data.transforms.KShots(valid_dataset, 2*shots),
+        l2l.data.transforms.LoadData(valid_dataset),
+        l2l.data.transforms.ConsecutiveLabels(train_dataset),
+        l2l.data.transforms.RemapLabels(valid_dataset),
+    ]
+    valid_tasks = l2l.data.TaskDataset(valid_dataset,
+                                       task_transforms=valid_transforms,
+                                       num_tasks=600)
+
+    test_transforms = [
+        l2l.data.transforms.NWays(test_dataset, ways),
+        l2l.data.transforms.KShots(test_dataset, 2*shots),
+        l2l.data.transforms.LoadData(test_dataset),
+        l2l.data.transforms.RemapLabels(test_dataset),
+        l2l.data.transforms.ConsecutiveLabels(train_dataset),
+    ]
```

Comment:
Is it not better to just have an import of the form:

`from l2l.data.transforms import NWays, KShots, LoadData, RemapLabels, ConsecutiveLabels`

and then directly use them?